### DISCUSSION TITLE
글 목록 중간에서 테마전환버튼/상단으로 버튼/사이드바 클릭 오류

### DISCUSSION THREAD

**Discussion by Nonlinear14** [pronist/hELLO#257] [Q&A] [answered]
안녕하세요 스킨 3버전때부터 너무나 애용중입니다.
다만 이번 4.10.0 스킨 업데이트 후에 다크/라이트 모드 전환 버튼과 상단으로 가기 버튼 오류가 있어서 해결법 아시는지 질문 드립니다.

![Screenshot_20241009_190605_Chrome](https://github.com/user-attachments/assets/b9779bce-6cd2-497d-9254-c2b6fdcbc70b)

사진에서처럼, 글 목록 중간에 테마 전환이나 상단가기 버튼을 누르면 테마 전환/상단가기 대신 해당 글이 열리는 오류가 있네요;^;
특히 모바일에서의 경우 화면 너비가 좁다보니 사이드바를 열고 카테고리를 클릭했을 때도 동일하게 해당 카테고리의 글 목록이 아닌, 중간에 걸쳐진 글의 링크가 열립니다.

html과 css의 경우에 해당 버튼 관련한 부분은 건드리지 않았고, 그 외의 스크립트와 소스코드 역시 별도로 수정하지는 않았습니다. 이전 버전인 4.9.0에서는 정상 작동해서 해당부분 html을 그대로 가져왔는데도 오류가 계속 있네요. 혹시 해결법이 있을까요?

**Comment by pronist** [comment, marked as answer]
@Nonlinear14 안녕하세요.

이 부분은 4.10 패치에서 커버를 건드리면서 생긴 버그입니다. 다음 패치때 수정이 예정되어 있습니다.
